src/steamship/client/client.py

```diff
@@ -28,6 +28,7 @@ def __init__(
         space_id: str = None,
         space_handle: str = None,
         create_space: bool = False,
+        fetch_or_create_space: bool = False,
```

Comment:
Isn't this what you'd want as default?  If I provide a space_handle, I expect to re-use or create a space. I'm guessing most of our audience would turn this param on as otherwise, the code cannot be run more than once.